docker/pulsar/scripts/install-pulsar-client.sh

```diff
@@ -23,3 +23,7 @@ set -x
 PYTHON_MAJOR_MINOR=$(python3 -V | sed -E 's/.* ([[:digit:]]+)\.([[:digit:]]+).*/\1\2/')
 WHEEL_FILE=$(ls /pulsar/pulsar-client | grep "cp${PYTHON_MAJOR_MINOR}")
 pip3 install /pulsar/pulsar-client/${WHEEL_FILE}[all]
+
+# TODO: remove these lines once grpcio doesn't need to compile from source on ARM64 platform
+apt update
+apt -y install build-essential python3-dev
```

Comment:
This would be increasing the size of the pulsar image.

I could see 2 different options to avoid it: 
 1. Only install compilers on ARM64 build (at least we don't make the x86_64 image any worse)
 2. Do the Grpc `pip install` in a separate Docker stage and then only copy the built wheel file. (this should work for both arm & x86)

Reply:
Resolved by option 1 at 279388542fe93df2b5dc4fc9f9986fb2c303c95a.

Option 2 seems also to require installing` build-essential python3-dev` for `pip install grpcio` on ARM64 platform? Conditional installation is easier to perform inside a script than inside the Dockerfile AFAIK...